wso2is_analytics/hieradata/dev/wso2/wso2is_analytics/pattern-1/README.md

```diff
@@ -0,0 +1,168 @@
+    
+    Datasource used Analytics record store. In default configurations analytics event store and processed data store 
+    is not separated out and a single datasource is configured and used for both. If needed two databases can be 
+    defined and two datasources can be configured as in [minimum HA deployment guideline](https://docs.wso2.com/display/IS540/Setting+Up+Deployment+Pattern+2#SettingUpDeploymentPattern2-MinimumHighAvailabilityDeploymentforWSO2ISAnalytics). 
+    e.g:
+    ```
+    wso2::analytics_datasources:
+      analytics_store_ds:
+        name: WSO2_ANALYTICS_STORE_DS
+        description: The datasource used for analytics record store
+        driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
+        url: jdbc:mysql://192.168.100.1:3306/WSO2_ANALYTICS_DB?autoReconnect=true&amp;useSSL=false
+        username: "%{hiera('wso2::datasources::mysql::username')}"
+        password: "%{hiera('wso2::datasources::mysql::password')}"
+        max_active: "%{hiera('wso2::datasources::common::max_active')}"
+        max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
+        test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
+        default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
+        validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
+        validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
+        default_auto_commit: false
+        initial_size: 0
+        test_while_idle: true
+        min_evictable_idle_time_millis: 4000
+        
+     # Analytics event store datasource configuration in repository/conf/analytics/analytics-config.xml
+     wso2::analytics_event_store_datasource: analytics_store_ds
+     
+     # Analytics processed data store datasource configuration in repository/conf/analytics/analytics-config.xml
+     wso2::analytics_processed_data_store_datasource: analytics_store_ds
+    ```
+4. Configure a clustering scheme for minimum HA deployment
+ 
+    In default Hiera configurations clustering is disabled. If you need to do a minimum HA deployment of WSO2 
+    Identity Server Analytics rather than a single node deployment you will have to enable clustering. 
+    For that, set ```enable``` property under ```wso2::clustering``` to ```truee```. Then, configure a clustering 
+    scheme under ```wso2::clustering```. By default, WKA related configurations are added. 
+    Update the member list under WKA scheme in [default.yaml](default.yaml) file with the ip addresses of the members
+    participating in the cluster.
+    e.g:
+    ```
+    wso2::clustering:
+    enabled: true
+    domain: is.analytics.wso2.domain
+    local_member_host: "%{::ipaddress}"
+    local_member_port: 4000
+    sub_domain: mgt
+    
+    membership_scheme: wka
+    wka:
+     members:
+       -
+         hostname: 192.168.48.194
```

Comment:
why %{::ipaddress} not used here ?

Reply:
This is the read me entry which specifically ask to configure nodes participating in the cluster

Reply:
Still, local member ip can be obtained from ipaddress, so, better to keep it here as it is in the hiera yaml.